mapeamento/mapeadores/spiders/mapeador_adiariosv1.py

```diff
@@ -0,0 +1,93 @@
+import datetime
+
+import scrapy
+
+from mapeadores.spiders.bases.mapeador_semantico import MapeadorSemantico
+from mapeadores.items import MapeamentoItem
+
+class MapeadorAdiarioV1(MapeadorSemantico):
+    name = "mapeador_adiariov1"
+
+    def pattern_name(self):
+        return "ADIARIOS_v1"
```

Comment:
O nome do raspador não poderia ser `adiariov1`? E se esse `pattern_name` realmente for útil, poderia ser uma property da classe `Mapeador` que devolve `name.upper`. O que acha?

Reply:
Boa, adotado também, o método `pattern_name()` foi removido. Ao invés de `name.upper`, adotei apenas `self.name` mesmo. Uso o `adiariov1` para imprimir a mensagem de progresso e também para registrar qual é o padrão mapeado no arquivo de saída.

Os arquivos de saída são separados por padrão, então pode ficar redundante ter esse campo no registro, mas pensando que um arquivo `mapeamento_completo` pode ser necessário, e varios padrões vão se misturar ali, saber que padrão é cada registro é útil.

![image](https://github.com/user-attachments/assets/432b545d-7708-40ab-9050-a04e8405e8a7)